popup/popup.js

```
@@ -1,146 +1,375 @@
+      });
+      showMessage(`API key storage set to: ${sessionOnlyCheckbox.checked ? "Session Only" : "Persistent"}`);
+    });
+  }
+  // Reset API counter
+  if (resetApiCountBtn) {
+    resetApiCountBtn.addEventListener('click', () => {
+      if (confirm("Reset API call counter to zero?")) {
+        setButtonLoading(resetApiCountBtn, true, "Resetting...");
+        chrome.runtime.sendMessage({action: "resetApiCount"}, (response) => {
+          setButtonLoading(resetApiCountBtn, false, "Reset Counter");
+          if (response && response.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)